### PR TITLE
[XPU] support op builder from intel_extension_for_pytorch kernel path

### PR DESCRIPTION
**Pull request by YizhouZ**
#Motivation
From our next release, xpu DeepSpeed related kernels would be put into intel_extension_for_pytorch. This PR is to add new op builders and use kernel path from intel_extension_for_pytorch. More ops like MOE and WOQ will be added.

### CODE REVIEW

**Comment by YizhouZ** [comment]
@loadams @mrwyattii Hi, could you help to trigger a CI for this PR? thanks!

**Comment by loadams** [comment]
> @loadams @mrwyattii Hi, could you help to trigger a CI for this PR? thanks!

Done @YizhouZ - could you run the pre-commit formatter to pass the formatting check?  Thanks

**Comment by YizhouZ** [comment]
> > @loadams @mrwyattii Hi, could you help to trigger a CI for this PR? thanks!
> 
> Done @YizhouZ - could you run the pre-commit formatter to pass the formatting check? Thanks

Fixed! Could you run it again? Thanks.

File: accelerator/xpu_accelerator.py [hunk 1]
```diff
@@ -253,33 +257,29 @@ def on_accelerator(self, tensor):
...
+
     # create an instance of op builder and return, name specified by class_name
-    def create_op_builder(self, op_name):
-        builder_class = self.get_op_builder(op_name)
-        if builder_class != None:
-            return builder_class()
-        return None
+    def create_op_builder(self, class_name):
+        builder_class = self.get_op_builder(class_name)
+        return None if builder_class is None else builder_class()
```
Comment (rogerxfeng8) [inline accelerator/xpu_accelerator.py:274]: builder_class could return NotImplementedBuilder or None. It's error prone to  check between the two.

**Comment by YizhouZ** [comment]
@tjruwase @loadams It seems v100 unit tests failed somehow in today's merging commit. I did not see any error msg in log, is it something related to CI server?

**Comment by loadams** [comment]
> @tjruwase @loadams It seems _nv-torch-latest-v100 / unit-tests_ tests failed somehow in today's merging commit. I did not see any error msg in log, is it something related to CI server?

@YizhouZ - this looks to be a transient error on the CI nodes, it should be fixed now.

**Comment by YizhouZ** [comment]
> > @tjruwase @loadams It seems _nv-torch-latest-v100 / unit-tests_ tests failed somehow in today's merging commit. I did not see any error msg in log, is it something related to CI server?
> 
> @YizhouZ - this looks to be a transient error on the CI nodes, it should be fixed now.

thanks! could you help to trigger CI?

**Comment by YizhouZ** [comment]
Thanks for triggering CI. Do you have comments on this PR? @loadams @tjruwase

**Comment by YizhouZ** [comment]
> Thanks for triggering CI. Do you have comments on this PR? @loadams @tjruwase

Hi @loadams @tjruwase, this PR seems not in the merge queue. Could you give us some suggestions for this PR? Thanks!

**Comment by loadams** [comment]
> > Thanks for triggering CI. Do you have comments on this PR? @loadams @tjruwase
> 
> Hi @loadams @tjruwase, this PR seems not in the merge queue. Could you give us some suggestions for this PR? Thanks!

Hi @YizhouZ - we just haven't had time to review yet, we will prioritize this.

File: accelerator/xpu_accelerator.py (hunk 1 above)
Comment (loadams) [reply accelerator/xpu_accelerator.py:275]: @YizhouZ thoguhts on this?

Comment (YizhouZ) [reply accelerator/xpu_accelerator.py:275]: > @YizhouZ thoguhts on this?

Previously we didn't add NoneImpl builder. Will remove this line to only return NoneImpl builder.

**Review by tjruwase — APPROVED** [review]

**Comment by YizhouZ** [comment]
Could you please add this PR into merge queue? Thanks! @tjruwase @loadams

**Pull request merged**